Dockerfile

```diff
@@ -9,10 +9,15 @@ RUN apt-get update && apt-get install -y \
   gcc \
   make \
   python3
+
+RUN yarn config set strict-ssl false
+ENV NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
FYI - I think this is necessary internally within ABSA, not sure if we should include in this repo?